server/pulp/server/async/scheduler.py

```diff
@@ -368,3 +369,6 @@ def add(self, **kwargs):
         entries to the database, and they will be picked up automatically.
         """
         raise NotImplementedError
+
+    def close(self):
```

Comment:
We call `super()` here. It may not do much but since we are overriding it and we didn't before we should preserve to codepath.